Added some sound effects

CarlosTFS opened the pull request:
Effects for dashing and shooting added

CarlosTFS committed:
Added some sound effects

Effects for dashing and shooting added

Helcostr commented:
@CarlosTFS please try and resolve the conflicts between the main's PlayerController.cs and your PlayerController.cs

edit: wait it was a simple merge. nvm

Helcostr committed:
Merge branch 'main' into Soundeffects

cothon01 committed:
Merge branch 'main' into Soundeffects

cothon01 commented:
Manually fixed a tiny merge conflict with the level scene (I kept the main branch changes)

cothon01 committed:
Minor Fixes

- Fixed a bug where shotgun doesn't shoot
- Fixed a small bug where the audiosource reference is null

cothon01 reviewed (approved):
Gun sounds and dash sound works
